mParticle-Kochava/MPKitKochava.m

```diff
@@ -2,11 +2,17 @@
 #import "MPKochavaSpatialCoordinate.h"
 #import "mParticle.h"
 #import "MPKitRegister.h"
+#import "MPIConstants.h"
```

Comment:
This breaks SPM since this file is not public. Not sure what this was being used for, maybe MPEnums.h would do?